codespell_lib/data/dictionary.txt

```diff
@@ -54374,6 +54374,7 @@ siplifies->simplifies
 siplify->simplify
 siplifying->simplifying
 siply->simply
+sipport->support
```

Comment:
I suggest adding all verbal forms:
```suggestion
sipport->support
sipported->supported
sipporting->supporting
sipports->supports
```

Reply:
agree! (github wouldn't let me commit this directly from the web interface for some reason)

Reply:
It's a GitHub bug, the interface fails for large files.